docs/source/60_building_apps.md

```diff
@@ -256,6 +256,58 @@ $ spark-submit \
 A complete example of this pattern can be found in the
 [heuermh/adam-examples](https://github.com/heuermh/adam-examples) repository.
 
+### Using ADAM’s RegionJoin API
+Another useful API implemented in ADAM is the RegionJoin API, which joins two
+genomic datasets that contain overlapping regions. This primitive is useful for
+a number of applications including variant calling (identifying all of the reads 
+that overlap a candidate variant), coverage analysis (determining the coverage 
+depth for each region in a reference), and INDEL realignment (identify INDELs 
+aligned against a reference).
+
+There are two overlap join implementations available in ADAM: 
+BroadcastRegionJoin and ShuffleRegionJoin. The result of a ShuffleRegionJoin 
+is identical to the BroadcastRegionJoin, however they serve different 
+purposes depending on the content of the two datasets.
+
+The ShuffleRegionJoin is at its core a distributed sort-merge overlap join. 
+To ensure that the data is appropriately colocated, we perform a copartition 
+on the right dataset before the each node conducts the join locally. The 
+BroadcastRegionJoin performs an overlap join by broadcasting a copy of the 
+entire left dataset to each node. ShuffleRegionJoin should be used if the right 
+dataset is too large to send to all nodes and both datasets have low 
+cardinality. The BroadcastRegionJoin should be used when you are joining a 
+smaller dataset to a larger one and/or the datasets in the join have high 
+cardinality.
+
+To perform a ShuffleRegionJoin, add the following to your ADAM script:
+
+```dataset1.shuffleRegionJoin(dataset2)```
+
+To perform a BroadcastRegionJoin, add the following to your ADAM script:
+
+```dataset1.broadcastRegionJoin(dataset2)```
+
+Where dataset1 and dataset2 are GenomicRDDs. If you used the ADAMContext to 
+read a genomic dataset into memory, this condition is met.
+
+ADAM has a variety of ShuffleRegionJoin types that you can perform on your 
+data, and all are called in a similar way:
+
+![Joins Available]
+(https://github.com/bigdatagenomics/adam/tree/master/docs/source/img/join_examples.png)
+
+
+Join call | action |
```

Comment:
Table looks good! Can we add a column that lists whether the action is supported in shuffle, broadcast, or both?